Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,336 @@
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+    requests.packages.urllib3.disable_warnings()
+
+    # hexdump:732073756468656572206b756d6172
+
+    class SymantecAuth(AuthBase):
+        def __init__(self, user, password, host):
+            self.basic = HTTPBasicAuth(user, password)
+            self.host = host
+
+        def __call__(self, r):
+            if r.url.startswith(self.host):
+                return self.basic(r)
+            else:
+                return r
+
+    SERVER_IP = demisto.params()['server-ip']
+    USERNAME = demisto.params()['credentials']['identifier']
+    PASSWORD = demisto.params()['credentials']['password']
+    SAVED_REPORT_ID = demisto.params()['saved-Report-id']
+    FETCH_TIME = demisto.params()['fetch_time']
+    SEVERITY = {'high':3,'medium':2,'low':1,'info':5}
+
+    wsdl = 'https://{}/ProtectManager/services/v2011/incidents?wsdl'.format(SERVER_IP)
+
+    session = Session()
+    session.auth = SymantecAuth(USERNAME,PASSWORD, "https://{}".format(SERVER_IP))
+    session.verify = False
+    cache = SqliteCache(timeout=None)
+    transport = Transport(session=session, cache=cache)
+
+    client = Client(wsdl=wsdl, transport=transport)
+
+    def  get_incident_id(dtobj):
+
+        if type(dtobj) is datetime:
+            return client.service.incidentList(savedReportId=SAVED_REPORT_ID, incidentCreationDateLaterThan=dtobj)
+        else:
+            return_error('excpected datetime object')
+
+
+
+    def incidentDetail(_id):
+
+        return client.service.incidentDetail(incidentId=_id,includeViolations=True,includeHistory=True)
+
+
+    def incidentBinaries(args):
+
+        return client.service.incidentBinaries(**args)
+
+
+    def myconverter(o):
+        if isinstance(o, datetime):
+            return o.__str__()
+
+
+    def fetchincidents():
+        last_run=demisto.getLastRun()
+        if 'lt_time' not in last_run:
+            fetch_time,_ = parse_date_range(FETCH_TIME)
+            last_run['lt_time'] = fetch_time
+        else:
+            last_run['lt_time']= datetime.strptime(last_run['lt_time'],'%d/%m/%Y %H:%M:%S.%f')
+
+        incidentids=get_incident_id(last_run['lt_time'])
+        incidents=[]
+        if len(incidentids['incidentId']) > 0:
+            for incident1 in incidentids['incidentId']:
+                inc=incidentDetail(str(incident1))
+                serialized_obj = helpers.serialize_object(inc[0])
+                incidents.append({
+                    'name':"#"+str(incident1) + " DLP "+ inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated",
+                    'details': json.dumps(serialized_obj,default = myconverter),
+                    'rawJSON': json.dumps(serialized_obj,default = myconverter),
+                    'severity' : SEVERITY[inc[0]['incident']['severity']]
+                })
+                if inc[0]['incident']['incidentCreationDate'].isoformat() > last_run['lt_time'].isoformat():
```

Comment:
No reason to do this `last_run['lt_time'].isoformat()` for every incident
Initialize it once and use it then